### ISSUE TITLE
No desktop

### ISSUE THREAD

**Issue by slde-gorillaman** [zyga/vagrant-desktop-images#14]
Hi there,

I was able to create the box and do a vagrant up, however I am stuck in the shell.
No desktop is being started, or did I misunderstand this repo?

**Comment by zyga** [comment]
Hey.

Which release did you try? It takes significant amount of time to actually
build the final desktop image. After that is done and you've saved it you
should be able to get it to work reliably. Note that building process is
headless (no desktop window will pop-up). Look at the tests/ directory to
see example vagrant files that do bring a window up for testing. You can
follow that as an example on what to do to use that in your configurations.

Thanks
ZK

2015-07-24 17:02 GMT+02:00 Khair-ed-Din Husseini <notifications@github.com>:

**Comment by slde-gorillaman** [comment]
I tried the master. The built of the box worked like a charme but when I used the sample from the readme as my Vagrant file I would get the ViritualBox window but only shell.
The Vagrantfile from the test did not work at all, as it tried to create a precise image instead of a trusty

**Comment by zyga** [comment]
Hmm, maybe you got a buggy build. I suspect there are quite a few flaws in
the puppet-based provisioner (originally it was all just plain shell that
always worked).

Try destroying the image and rebuilding. I've uploaded a correct 12.04
image to vagrant cloud. The 14.04 image is buggy (screensaver settings are
not applied) so I did not upload it. If you have the time to work on this
and fix it I would gladly accept patches and upload a fixed box for
everyone to consume.

2015-07-24 17:50 GMT+02:00 Khair-ed-Din Husseini <notifications@github.com>:
